File: Difficulty: Easy/Missing in Array/missing-in-array.py
#User function Template for python3
class Solution:
    def missingNumber(self, arr):
        n=len(arr)+1
        ts=n*(n+1)//2
        s=sum(arr)
        return ts-s
#{ 
 # Driver Code Starts
#Initial Template for Python 3

t = int(input())
for _ in range(0, t):
    arr = list(map(int, input().split()))
    s = Solution().missingNumber(arr)
    print(s)

    print("~")
# } Driver Code Ends

File: Difficulty: Easy/Missing in Array/missing-in-array.cpp
class Solution {
  public:
    int missingNum(vector<int>& arr) {
        // code here
        int n=arr.size()+1;
        int totals=0,s=0;
        for(int i=1;i<=n;i++){
            totals+=i;
        }
        for(int it:arr){
            s+=it;
        }
        return totals-s;
    }
};